adodbapi/readme.txt

```diff
@@ -22,14 +21,9 @@ Features:
 Prerequisites:
 * C Python 2.7 or 3.5 or higher
  and pywin32 (Mark Hammond's python for windows extensions.)
-or
- Iron Python 2.7 or higher.  (works in IPy2.0 for all data types except BUFFER)
 
 Installation:
 * (C-Python on Windows): Install pywin32 ("pip install pywin32") which includes adodbapi.
-* (IronPython on Windows): Download adodbapi from http://sf.net/projects/adodbapi.  Unpack the zip.

```

Comment:
Maybe leave the reference to the old sf.net source so that IronPython users will still have a hint where to find it.